store-failures

```diff
@@ -138,6 +138,11 @@ def find_failed_tests(fp):
             save_message = False
             continue
 
+        # If the issue is known, don't save unexpected messages
+        if line.startswith("ok ") and "SKIP Known issue" in line:
```

Comment:
Technically the "SKIP Known issue" is redundant IMHO. If we consider the test as passed, then we shouldn't second-guess it here.

Reply:
Yeah, I was just not sure, if `ok ` is strong enough patter and it could not appear as part of something else as well.

Reply:
Oh, good point - this isn't the last line of the test only, it's "anywhere in the log". So indeed, let's leave it as-is.

Reply:
Oh, good point - this isn't the last line of the test only, it's "anywhere in the log". So indeed, let's leave it as-is.